docs/development.md

```diff
@@ -403,6 +403,7 @@ You can optionally set the following variables:
 | `SKIP_CREATE_MGMT_CLUSTER` | Skip management cluster creation. If skipping managment cluster creation you must specify `KUBECONFIG` and `SKIP_CLEANUP` | `false`                                                                               |
 | `LOCAL_ONLY`               | Use Kind local registry and run the subset of tests which don't require a remotely pushed controller image.               | `true`                                                                                |
 | `REGISTRY`                 | Registry to push the controller image.                                                                                    | `capzci.azurecr.io/ci-e2e`                                                            |
+| `CLUSTER_NAME`             | Name of an exisiting workload cluster.  Will run specs against existing workload cluster. Use in conjunction with `SKIP_CREATE_MGMT_CLUSTER`, `GINKGO_FOCUS` and `KUBECONFIG`. Must specify only one e2e spec to run against with `GINKGO_FOCUS` such as `export GINKO_FOCUS=Creating.a.VMSS.cluster.with.a.single.control.plane.node`. | `capz-e2e-uf9yll`                                       |
```

Comment:
what's `capz-e2e-uf9yll` ? there shouldn't be a default for this

Reply:
will remove, thought it was examples but see now that it is defaults

Reply:
removed